Cargo.toml

```diff
@@ -11,6 +11,7 @@ readme = "README.md"
 license = "MIT/Apache-2.0"
 exclude = ["/ci/*"]
 edition = "2018"
+rust = "1.48.0"
```

Comment:
This is supposed to be `rust-version` and that only starts working after 1.56, so I don't think we should add it here.

Reply:
Deleted. thanks!

Reply:
So please squash this into the first commit?

Reply:
Squashed.